web/static/js/configs/stage_configs.jsx

```diff
@@ -21,6 +21,10 @@ const baseIdeaGenerationConfig = {
     headerText: "Stage Change: Idea Generation!",
     BodyComponent: StageChangeInfoIdeaGeneration,
   },
+  help: {
+    headerText: "Idea Generation",
+    BodyComponent: StageChangeInfoIdeaGeneration,
+  },
```

Comment:
@vanderhoop There were a few ways to solve the "Don't show the Stage Change!" text requirement, I chose to introduce some duplication in the config object instead of some sort of conditional in the component.  Pros and cons with each approach, but I felt was the cleanest.

Reply:
yeah, i hear that. think this is a sensible choice.